ipywidgets/widgets/widget_media.py

```diff
@@ -165,6 +165,49 @@ def __repr__(self):
         return self._get_repr(Image)
 
 
+class MapArea(object):
+    r"""
+    An <area> tag, part of an image map <map>.
+    """
+    def __init__(self, name, shape, coords, href=None):
+        """
+        Parameters
+        ----------
+        name: str
+            Area name. Will be used as 'alt' attribute.
+
+        shape: str
+            Area shape: rect/circle/poly/default.
+
+        coords: tuple
+            Area coordinates: tuple of integers.
+
+        href: str
+            URL to load when area is clicked.
+        """
+        self.name = name
+        self.shape = shape
+        self.coords = coords # a tuple of integers
+        self.href = href
```

Comment:
I wonder if `href` is what we need, maybe an `on_click` event would be easier to play with for users in Python?

Reply:
Similar to the Button API